apps/dashboard/app/views/batch_connect/sessions/card/_display_choices.html.erb

```diff
@@ -0,0 +1,7 @@
+<% user_context = session.user_context %>
+<% session.app.attributes.select(&:display?).map do |attribute| %>
```

Comment:
I think you should do `each` here instead of `map`.

`map` has a return value (a useless one in this case) where `each` does not (meaning map makes a copy).